dist_tables/append_distribution.rst

```diff
@@ -141,7 +139,19 @@ The example below deletes those shards from the github_events table which have a
                                3
     (1 row)
 
-To learn more about the function, its arguments and its usage, please visit the :ref:`user_defined_functions` section of our documentation.
+To learn more about the function, its arguments and its usage, please visit the :ref:`user_defined_functions` section of our documentation.  Please note that this function only deletes complete shards and not individual rows from shards. If your use case requires deletion of individual rows in real-time, see the section below about deleting data.
+
+Deleting Data
+---------------
+
+The most flexible way to modify or delete rows throughout a Citus cluster is the master_modify_multiple_shards command. It takes a regular SQL statement as argument and runs it on all workers:
+
+::
+
+  SELECT master_modify_multiple_shards(
+    'DELETE FROM github_events WHERE created_at <= ''2014-01-01 00:00:00''');
+
+This uses a two-phase commit to remove or update data safely everywhere. Unlike master_apply_delete_command, it works at the row- rather than shard-level to modify or delete all rows that match the condition in the where clause. It deletes rows regardless of whether they comprise an entire shard. To learn more about the function, its arguments and its usage, please visit the :ref:`user_defined_functions` section of our documentation.
```

Comment:
2PC isn't the default. So we could say something like: "The function uses a configurable commit protocol to update or delete data safely across multiple shards."